### PR TITLE
Added diagnostic_aggregator and updated config.

### PR DESCRIPTION
**Pull request by tonybaltovski**
@mikepurvis

### CODE REVIEW

File: ridgeback_base/launch/base.launch [hunk 1]
```diff
@@ -14,7 +14,14 @@
 
   <node pkg="ridgeback_base" type="ridgeback_node" name="ridgeback_node" output="screen" />
 
-  <!-- Differential controller and basic localization -->
+  <!-- Mecanum controller and basic localization -->
   <include file="$(find ridgeback_control)/launch/control.launch" />
 
+  <!-- Teleoperation via joystick and rviz interactive markers -->
+  <include file="$(find ridgeback_control)/launch/teleop.launch" />
```
Comment (mikepurvis) [inline ridgeback_base/launch/base.launch:21]: Was this previously not being run somewhere else?

Comment (tonybaltovski) [inline ridgeback_base/launch/base.launch:21]: There was a launch file in the robot_upstart directory.

Comment (mikepurvis) [inline ridgeback_base/launch/base.launch:21]: Cool, sounds good.

**Comment by mikepurvis** [comment]
LGTM.

**Pull request merged**